### ISSUE TITLE
Gettin build error

### ISSUE THREAD

**Issue by nutan06** [joeldenning/single-spa-examples#44]
I am getting build error while running the single-spa-examples code.
The Webpack example runs nicely but getting issue when 2 versions of angular are added.
Can you explain a bit on single-spa-examples or how to use multiple versions of angular in 
simple-single-spa-webpack-example. I didnt get how to create seperate package.json.

Thanks in advance

**Comment by internettrans** [comment]
What error are you getting building the code?

Here are a few different ways two pull off separate package.json files for each child application:
- Publish each child app to npm and have the root app npm install them.
- Have each child app have its own package.json and webpack.config.js. Host each of the bundles on a web server and have SystemJS load them from the browser.

Let me know if you'd like more details on any of that.

Somewhat related, and maybe helpful:
- https://github.com/CanopyTax/single-spa/issues/123
- https://github.com/CanopyTax/single-spa/issues/102

**Comment by nutan06** [comment]
Thank you for the quick respnse. I will try with seperate package.json as mentioned.After downloading the single-spa-example if I try to run it I get following error

![image](https://user-images.githubusercontent.com/10524403/32465722-4920ee5c-c2f9-11e7-9b39-59aa172e4739.png)

**Comment by internettrans** [comment]
I was able to reproduce this -- it looks like you might not have run `npm install` before running `npm start` and opening it up in the browser. See https://github.com/CanopyTax/single-spa-examples#how-to-get-the-examples-running-locally for the instructions. When you do `npm install`, it should automatically create a `build` directory and create all of those files that are not being found in your stacktrace. The way it is doing it is [with a postinstall hook here](https://github.com/CanopyTax/single-spa-examples/blob/master/package.json#L5). If for some reason, that post install hook is not running after you do `npm install`, you can manually trigger it by doing `npm run postinstall` after you do npm install.

Try this:
```bash
npm install
npm run postinstall # Should get run automatically at the end of install, but just in case it isn't you can do it manually like this.
npm start
# Now open up http://localhost:8080 in the browser
```

**Comment by nutan06** [comment]
Will try this .Thank you very much for prompt response. I appreciate this.

**Comment by nutan06** [comment]
Followed all the instructions above. Now I am getting 404 error.

![image](https://user-images.githubusercontent.com/10524403/32566910-69465a66-c46e-11e7-98b6-cfa2e13171e7.png)

**Comment by internettrans** [comment]
Can you verify if you have a `build` directory? It should have been created during `npm run postinstall`. Here's what my single-spa-examples repo looks like after doing the npm install (notice the build directory):

![screen shot 2017-11-08 at 11 26 25 am](https://user-images.githubusercontent.com/5524384/32567199-cf43200c-c477-11e7-810c-769fd9e660d0.png)

If you do not have a build directory, your `npm install` or `npm run postinstall` must have failed. Try running `npm run build` to see if it creates the build directory. If you get any errors in the terminal running those commands, let me know because that would prevent the build directory from being created properly.

**Comment by joeldenning** [comment]
Can you verify if you have a `build` directory? It should have been created during `npm run postinstall`. Here's what my single-spa-examples repo looks like after doing the npm install (notice the build directory):

![screen shot 2017-11-08 at 11 26 25 am](https://user-images.githubusercontent.com/5524384/32567199-cf43200c-c477-11e7-810c-769fd9e660d0.png)

If you do not have a build directory, your `npm install` or `npm run postinstall` must have failed. Try running `npm run build` to see if it creates the build directory. If you get any errors in the terminal running those commands, let me know because that would prevent the build directory from being created properly.

**Comment by nutan06** [comment]
Yes I do have similar thing but I see following error in my cmd prompt.

![image](https://user-images.githubusercontent.com/10524403/32570657-ae8b45c2-c479-11e7-8a6b-b59bb5d2b034.png)

Can you get what this error is.I am new to jspm and trying to understand it.

**Comment by nutan06** [comment]
This is what I get after npm install

![image](https://user-images.githubusercontent.com/10524403/32570804-25518e00-c47a-11e7-8026-ca290d889d6e.png)

**Comment by nutan06** [comment]
@joeldenning got the issue,I am running this on windows . Facing similar issues as mentioned below:
[https://github.com/CanopyTax/single-spa-examples/issues/40](url) 
It worked when tried on mac.
So how do I rectify this in windows?

**Comment by internettrans** [comment]
@nutan06 apologies for the (very) slow response, but this should now be fixed (see #40). Closing this for now, let me know if you still have issues!
